application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReloadModMailCommand.java

```diff
@@ -0,0 +1,45 @@
+package org.togetherjava.tjbot.commands.moderation;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.commands.SlashCommandVisibility;
+import org.togetherjava.tjbot.commands.modmail.ModMailUtil;
+import org.togetherjava.tjbot.config.Config;
+
+import java.util.Objects;
+
+public class ReloadModMailCommand extends SlashCommandAdapter {
+
+    private final JDA jda;
+    private static final Config config = Config.getInstance();
+
+    /**
+     * Creates an instance of ReloadMod command.
+     *
+     * @param jda the JDA instance to find the guild.
+     */
+    public ReloadModMailCommand(@NotNull JDA jda) {
+        super("reloadmod", "Reloads the list of moderators in the modmail selection menu",
+                SlashCommandVisibility.GUILD);
+
+        this.jda = Objects.requireNonNull(jda);
+    }
+
+    @Override
+    public void onSlashCommand(@NotNull SlashCommandEvent event) {
+        Guild guild = Objects.requireNonNull(jda.getGuildById(config.getGuildId()),
+                "A Guild is required to use this command. Perhaps the bot isn't on the guild yet");
```

Comment:
you should be able to do `event.getGuild()`, no?

Reply:
I could but I didn't due to some pedantic reason. If the command was called on Guild A, but the `guildId` was Guild B, then it would have inconsistent results.

Reply:
Thinking about that, I think I could perhaps do `event.getGuild()` on the `ModMailCommand` as well.

Reply:
I do think you should SlashCommandEvent#getGuild ^^

If you run the command in Guild B, right now it runs it like it's in Guild A, which is really weird imo

The bot is only in 1 guild, and it'll never join another one